core/domain/platform_feature_services_test.py

```diff
@@ -238,7 +238,7 @@ def test_update_feature_flag_rules_successfully_updates_rules(self):
                             'conditions': [
                                 [
                                     '=',
-                                    platform_parameter_domain.FEATURE_STAGES.dev
```

Comment:
This is technically a different enum value and should be managed as a separate constant (it's part of FEATURE_STAGES not SERVER_MODES).

Reply:
Yeah, makes sense. WIll change it :)

Reply:
done